src/main/java/com/techiteasy/demo/models/Television.java

```diff
@@ -0,0 +1,202 @@
+package com.techiteasy.demo.models;
+//De 'models' vertegenwoordigen de data-objecten die worden opgeslagen in de database. Deze klasse kan gebruikt worden om een tabel te genereren in de database via JPA.
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.time.LocalDate;
+
+//@Entity markeert deze klasse als een JPA-entiteit (wat betekent dat deze overeenkomst met een tabel in de database)
+@Entity
+public class Television {
+
+    @Id //Dit attribuut geeft aan dat het veld 'id' de primaire sleutel is.
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    //Deze annotatie zorgt ervoor dat de waarde van 'id' automatisch wordt gegenereerd door de database wanneer een nieuwe rij wordt ingevoegd
+    private long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    private LocalDate purchaseDate;
+    private LocalDate saleDate;
+    private Double availableSize;
+    private Integer refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambilight;
+    private Integer originalStock;
+    private Integer sold;
+
+    //Default constructor - dit is handig wanneer je een object wilt aanmaken zonder onmiddellijk waarden toe te wijzen aan de velden.
+    public Television() {
```

Comment:
Heel goed, als je een constructor met parameters maakt, is het vaak handig om er ook een constructor zonder parameters naast te maken. Dit is namelijk de default constructor, maar de default constructor vervalt als je zelf een andere constructor maakt.